### DISCUSSION TITLE
Need to organize alert rules under respective folder-name and only view the alert rules under respective folder-name by clicking the expansion button

### DISCUSSION THREAD

**Discussion by harish422** [grafana/grafana#67896] [Alerting]
### 
# What went wrong?

**What happened**:

  - Current grafana "Alert-Rules" page lists all the alerts with a folder name referencing the alert rule. This is adding unnecessary complexity since it becomes hard to read this page when there are lots of alerts configured under many folders.  
  
  
![Screen Shot 2023-05-03 at 12 15 37 PM](https://user-images.githubusercontent.com/23487589/236023227-b00bf91a-33e1-4b7a-8937-8c4af5d99901.png)

**What did you expect to happen**:

  - Instead if alerts can be expanded by clicking the folder name, that makes it much better to read. So no need to list all the alerts under all folders at all times. Users can expand the folder-name to view the alerts.
  - In above screen shot I should just expand "folder_EKS" to then see all the alerts. More like a parent-child hierarchy. 

### 
# How do we reproduce it?

Configure multiple alert rules under multiple folders. Then go to "Alert Rules" page to see all the alert rules listed in an un-organized fashion.

Need to organize them under respective folder-name and only view the alert rules under respective folder-name by clicking the expansion button

### 
# What Grafana version are you using?

9.4

**Comment by tonypowa** [comment]
Hi @harish422 

Thank you for your feedback. I am tagging the Alerting team for visibility. 

Since this is a feature request, not a bug, I am moving this conversation to GitHub discussions.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82683. Feel free to continue the discussion around this there. Thank you!
